src/python/WMCore/Services/DBS/DBS3Reader.py

```diff
@@ -592,8 +597,8 @@ def listFileBlockLocation(self, fileBlockName, dbsOnly = False, phedexNodes=Fals
                 msg += "%s\n" % formatEx3(ex)
                 raise DBSReaderError(msg)
 
-            if not any(blockInfo.values()): # no data location from dbs
-                return list()
+        if not any(blockInfo.values()): # no data location
```

Comment:
In principle, yes :) But it would be better to check what is calling this method (a few places) to make sure it would handle an empty list the same way as a dict with empty list values. Otherwise, just keep it as it was before and we take the safe side